### DISCUSSION TITLE
How can I use it with resolveConfig?

### DISCUSSION THREAD

**Discussion by utiq** [RyanClementsHax/tailwindcss-themer#39] [General]
I do 👇, to get my colors in the js code
```
import resolveConfig from "tailwindcss/resolveConfig";
import tailwindConfig from "../tailwind.config.js";
const fullConfig = resolveConfig(tailwindConfig);
```
how can I do with tailwindcss-themer?

or maybe, how can I change the theme from the JS code and update the fullconfig?

**Comment by RyanClementsHax** [comment]
Are you running into a problem doing it this way? (sorry for the late response)
